_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -43,15 +43,23 @@ Each test target's `autocomplete` [attribute value][] has a [correct autocomplet
 
 ## Assumptions
 
-The `autocomplete` attribute is not used on form fields that do not correspond to an autocomplete field described in the HTML 5.2 specification. If the `autocomplete` field is used to describe "custom" taxonomy, rather than that described in the specification, success Criterion [1.3.5 Identify Input Purpose][sc135] may be satisfied even if this rule failed. 
+The `autocomplete` attribute is not used on form fields that:
+
+- do not correspond to [Input Purposes for User Interface Components](https://www.w3.org/TR/WCAG21/#input-purposes), and
+- do not collect information about the user.
+
+If the `autocomplete` field is used to describe "custom" taxonomy, rather than that described in the list of `input` purposes, or the form fields do not collect information about the user, success Criterion [1.3.5 Identify Input Purpose][sc135] may be satisfied even if this rule failed.
 
 The `type` attribute is used correctly according to the intended purpose of `input` elements. If an incorrect `type` attribute is used for `input` elements, this rule may fail elements that satisfy success Criterion [1.3.5 Identify Input Purpose][sc135]. For example if an `input` element has a `type` of `number`, but is expecting an e-mail address.
 
+The `aria-disabled` state is used on `input` elements which are not part of [sequential focus navigation][] and are not otherwise [operable](https://www.w3.org/TR/wai-aria-1.2/#dfn-operable). It means that this rule may fail elements which satisfy success Criterion [1.3.5 Identify Input Purpose][sc135].
+
 ## Accessibility Support
 
 - While `autocomplete` is a promising technique for supporting personalization in HTML, support for this in assistive technologies is fairly limited.
 - Implementation of [Presentational Roles Conflict Resolution][] varies from one browser or assistive technology to another. Depending on this, some elements can have a [semantic role][] of `none` and fail this rule with some technology but users of other technologies would not experience any accessibility issue.
 - Some user agents treat the value of the `aria-disabled` attribute as case-sensitive.
+- Test targets can have a [form owner](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#form-owner) with the `autocomplete` [attribute value][] set to `off`. In such a case, user agents may not autofill the form fields, but those elements are still programmatically identifiable.
```

Comment:
When you say "use agents *may not*", do you mean "the specs leave that possibility open, UA can do as they wish" (the normative sense of "may") or "the specs explicitly disallow this possibility" (the normative sense of "may not") or "some UA do it one way, some do it another way" (the usual thing we flag in Accessibility Support?

If it's one of the first two, it is a Background info, not an Accessibility Support. If it,s the third (as I understand from #1400 referring different behaviour in Chrome/Firefox), make it explicit.

Reply:
Thanks @Jym77!
There is no mention about this in the spec. As far as I can tell, the only word about `autocomplete="off"` on the form owner in the spec refers to how the default values are computed for elements which are missing the `autocomplete` attribute (link below).

https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofilling-form-controls:-the-autocomplete-attribute:attr-fe-autocomplete-7

You're right it refers to the inconsistencies. I'll change that too.

Reply:
https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion has some good explanations on how autocomplete functions in firefox for these different scenarios.

If I've understood correctly, `autocomplete="off" indicates a browser should not remember what is input to a field/form, and should not automatically fill in the field/form, but it may still suggest previously remembered values for a field.

I think this makes it background understanding, rather than an inconsistency of accessibility support.

Reply:
Thanks @EmmaJP!

It definitely needs re-writing as it causes confusion. I mixed two issues together.

The point about auto-filling/ autocompleting is, in my opinion, more of a side note.  You're right, it should go to the background section.

However, @Jym was right in saying that the accessibility support statement for the `autocomplete` property of `input` should be more explicit. As it stands Firefox returns an empty string whenever the `autocomplete` property is queried. Chrome and Edge, on the other hand, return the correct `autocomplete` value. This is an accessibility support issue as the `autocomplete` attribute in Firefox may not work for AT.